docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+find . \! -user orise -exec chown orise '{}' +
```

Comment:
What this line is doing exaclty?

Reply:
"from the current path, find all files that user is not 'orise' and exec for each one of them `chown orise`"

Reply:
making it simpler, it changes the ownership of everything inside the home folder to the user orise